api/configs/middleware/vdb/analyticdb_config.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class AnalyticdbConfig(BaseModel):
+    """
+    Analyticdb configs
+    """
+
+    ANALYTICDB_KEY_ID : Optional[str] = Field(
+        description='Analyticdb key_id',
```

Comment:
Hi, try to put the text in a more human-readable way for explanation and hints, rather than just copying the exact id of the field. Be nice, be informative.

Reply:
Done. Thank you.

Reply:
Thanks!